medusa/scene_exceptions.py

```diff
@@ -286,7 +286,7 @@ def _get_custom_exceptions(force):
             )
             try:
                 # When any Medusa Safe session exception, session returns None and then AttributeError when json()
-                jdata = indexerApi(indexer).session.get(location, timeout=60).json()
+                jdata = MedusaSafeSession().get(location, timeout=60).json()
```

Comment:
You can easily reuse session objects. Not really efficient to create a new one each time